fuel/datasets/base.py

```diff
@@ -28,10 +29,6 @@ class Dataset(object):
         'targets')`` for MNIST (regardless of which data the data stream
         actually requests). Any implementation of a dataset should set this
         attribute on the class (or at least before calling ``super``).
-    default_iteration_scheme : :class:`.IterationScheme`, optional
```

Comment:
I'll make it part of this one, uprooting everything anyway :/ hopefully for
the better.
On Feb 26, 2015 9:12 AM, "Dmitry Bogdanov" <notifications@github.com> wrote:

> In fuel/datasets/base.py
> <https://github.com/bartvm/fuel/pull/20#discussion_r25428637>:
>
> > @@ -28,10 +29,6 @@ class Dataset(object):
> >          'targets')`` for MNIST (regardless of which data the data stream
> >          actually requests). Any implementation of a dataset should set this
> >          attribute on the class (or at least before calling ``super``).
> > -    default_iteration_scheme : :class:`.IterationScheme`, optional
>
> Okay, I am 100% happy to have get_example_stream instead. Would you
> prefer to make it a part of this PR or of a later?
>
> class ExampleScheme(BatchScheme):
>
>     def __init__(self, num_examples):
>         super(ExampleScheme, self).__init__(num_examples, 1)
>
>     def get_request_iterator(self):
>         return _iter(range(self.num_examples))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bartvm/fuel/pull/20/files#r25428637>.
>